### ISSUE TITLE
Atypical gestures

### ISSUE THREAD

**Issue by bcoleman96** [obophenotype/human-phenotype-ontology#9025]
**Preferred term label:**
Atypical gestures

**Synonyms**

**Definition (free text, please give PubMed ID)**
PMID: 16911975
Spontaneous production of communicative gestures that are perceived as odd or mechanical, or are poorly integrated with other means of communication. This may include inaccurate mimicry of typical gestures.

**Parent term (use hpo.jax.org/app)**
HP:4000090

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-4422-1708

**Comment by pnrobinson** [comment]
added
